src/Microsoft.PowerShell.ConsoleHost/host/msh/UpdatesNotification.cs

```diff
@@ -126,6 +126,7 @@ internal static void ShowUpdateNotification(PSHostUserInterface hostUI)
 
                 string notificationMsg = string.Format(CultureInfo.CurrentCulture, notificationMsgTemplate, releaseTag, notificationColor, resetColor, line2Padding, line3Padding);
 
+                hostUI.WriteLine();
```

Comment:
```suggestion
```

Reply:
We want an empty line when we need to write out update notification message. Otherwise, no blank line. So, the existing change is desired.